product_docs/docs/efm/4/10_notifications.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Notifications"
-redirects: 
+redirects:
  - ../efm_user/10_notifications
```

Comment:
I don't know how to make my editor not do these troublesome whitespace "fixes," but this and a couple other places are just whitespace changes.